mariadb-clients.hcl

```diff
@@ -0,0 +1,10 @@
+description = "Mariadb client."
```

Comment:
Can you please rename it to `mariadb-client.hcl`, so it's consistent with the existing `mysql-client.hcl`.